README.md

```diff
@@ -39,10 +40,17 @@ We have much better [Donuts](#donut-shops)
  * We have Lyft and that other company also
  * Those orange bikes you see around town are available on-demand through the [BikeTown program](https://www.biketownpdx.com). You'll have to register but the experience is supposed to be pretty painless overall (helmets are not provided). Per their [User Agreement](https://www.biketownpdx.com/rental-agreement) - [email their legal team](mailto:legal@motivateco.com) with subject header "ARBITRATION AND CLASS ACTION WAIVER OPT-OUT" if you don't want to be bound to their arbitration rules. Also note that due to Biketown's relationship to the Nike company, the service is most commonly pronounced as "Bikey [rhymes with Nike] town".
  * There are Five Quadrants: Burnside divides North/South, Willamette and I-5 East/West. Our magical fifth quadrant is North Portland, which is North of the Willamete and West of I-5, but NE was already taken.
- * Couch is pronounced "cooch" and Oregon is "orygun"
  * In parts of town, streets running East/West are in alphabetical order. North/South Streets are numbered. It's easy to get around.
  * This is [Bridge Town](http://www.bizave.com/portland/bridges/).
 
+##### Pronunciation Guide
+* **Willamette** /wəˈlæmɪt/ "wuh-LA-mit", rhymes with 'dammit'
+* **Couch**      /kutʃ/     "cooch"
+* **Glisan**     /ˈglisən/  "GLEE-sin"
+* **Kearney**    /ˈkʌɹni/   "KUR-nee"
+* **Going**      /gɔɪŋ/     "goyng", rhymes with 'boing'
+* **Savior**     /ˈseɪvjəɹ/ "SAY-vyer", sounds like 'savior'
```

Comment:
Got one for Oregon (since you snipped the one on line 42?

Reply:
so I was gonna put that in, but then I had some thoughts.
1. I could put in MY pronunciation, what *I* think is ""correct"", but like.. I didn't grow up here.. ¯\_(ツ)_/¯ 
2. I couldn't figure out what "orygun" what supposed to really sound like, since that is not how I would write MY pronunciation, but, again ¯\_(ツ)_/¯

Reply:
so I'd be happy to add one, but I'd rather get a better pronunciation from someone, and then I can write the IPA 😝

Reply:
Oregon (/ˈɒrɪɡən/)

Reply:
that first vowel is not one I recognize from learning the vowels in (standard) American English.. 
and the /r/ is wrong, unless you roll them like 'rr' in Spanish. 😜

Reply:
I grabbed it from Wikipedia. You should argue with them. I can't read IPA.

Reply:
https://en.wiktionary.org/wiki/Oregon offers:

 /ˈɔɹɪɡən/, /ˈɔɹəɡɪn/, (proscribed) /ˈɔɹɪɡɑn/

Reply:
If the best we've got can be found on Wikipedia, then I feel like it's not worth adding here..

Reply:
Aaaaand Dictionary.com offers us /ˌɔr ɪˈgoʊ ni ən, ˌɒr‐/

Reply:
OK. We're leaving it off. let's let the visitors fight with the yocals and call it a night.